prow/jobs/kyma-project/keda-manager/keda-manager-build.yaml

```
@@ -174,7 +174,7 @@ postsubmits: # runs on main
               - "--context=."
               - "--dockerfile=Dockerfile"
               - "--tag={{ .Env \"PULL_BASE_SHA\" }}"
-              - "--tag=latest"
+              - "--tag={{ .Env \"PULL_BASE_REF\" }}"
```

This also applies for the line above
```suggestion
              - "--tag=$(PULL_BASE_REF)"
```